[2012-12-18 14:24:30] request.INFO: Matched route "_register" (parameters: "_controller": "CloudPod\UserBundle\Controller\RegistrationController::registerAction", "_route": "_register") [] []
[2012-12-18 14:24:30] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2012-12-18 14:24:32] request.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO reg_user (username, salt, password, email, is_Active, confirmation_code, created_At, fk_role_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?)' with params {"1":"teffi","2":"a1a9dedb616a99ddf5527195c6a8dd3a","3":"EPXveFNkwZDu\/q8iJY3noYxJReW2ZyEjswSQ0dhXEaAd7wDUSW\/HUHkrfUB8Agza6\/QeWaFKeMOGt49B7ULFpQ==","4":"iffets_07@hotmail.com","5":0,"6":"3d19ac84fca979bed4faa5d913654364","7":"2012-12-18 22:24:30","8":1}:

SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'teffi' for key 'UNIQ_51BC446BF85E0677' (uncaught exception) at C:\wamp\www\CloudPod\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php line 47 [] []
[2012-12-18 14:24:33] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-18 14:27:43] request.INFO: Matched route "_register" (parameters: "_controller": "CloudPod\UserBundle\Controller\RegistrationController::registerAction", "_route": "_register") [] []
[2012-12-18 14:27:44] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2012-12-18 14:27:45] request.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO reg_user (username, salt, password, email, is_Active, confirmation_code, created_At, fk_role_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?)' with params {"1":"teffi","2":"0a7d6d0c871ddae2229fd35503855ef1","3":"gbIavGcWIFtUPrTo38ajNCc0EcqEmAsiKazwtN6z8TtVC0TyE5QuLiIniPYi6Wd4RyV04D1UpQx8McfJ5VYdoQ==","4":"getaway_0123@yahoo.com","5":0,"6":"7e0abbad2914a57a743bc42968d6b692","7":"2012-12-18 22:27:44","8":1}:

SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'teffi' for key 'UNIQ_51BC446BF85E0677' (uncaught exception) at C:\wamp\www\CloudPod\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php line 47 [] []
[2012-12-18 14:27:45] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 09:48:28] request.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at C:\wamp\www\CloudPod\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 92 [] []
[2012-12-31 09:50:16] request.INFO: Matched route "_security_login" (parameters: "_controller": "CloudPod\UserBundle\Controller\SecurityController::loginAction", "_route": "_security_login") [] []
[2012-12-31 09:50:17] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2012-12-31 09:50:17] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:Forms:login.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 9238 [] []
[2012-12-31 09:50:17] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 10:06:02] request.INFO: Matched route "_security_login" (parameters: "_controller": "CloudPod\UserBundle\Controller\SecurityController::loginAction", "_route": "_security_login") [] []
[2012-12-31 10:06:02] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2012-12-31 10:06:02] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:Forms:login.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\vendor\twig\twig\lib\Twig\Template.php line 280 [] []
[2012-12-31 10:06:02] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 10:06:11] request.INFO: Matched route "_security_login" (parameters: "_controller": "CloudPod\UserBundle\Controller\SecurityController::loginAction", "_route": "_security_login") [] []
[2012-12-31 10:06:12] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2012-12-31 10:06:12] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:Forms:login.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\vendor\twig\twig\lib\Twig\Template.php line 280 [] []
[2012-12-31 10:06:12] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 10:07:30] request.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 5110 [] []
[2012-12-31 10:07:49] request.INFO: Matched route "_user_profile" (parameters: "_controller": "CloudPod\UserBundle\Controller\UserController::profileAction", "id": "2", "_route": "_user_profile") [] []
[2012-12-31 10:07:49] security.DEBUG: Read SecurityContext from the session [] []
[2012-12-31 10:07:49] security.DEBUG: Reloading user from user provider. [] []
[2012-12-31 10:07:50] security.DEBUG: Username "" was reloaded from user provider. [] []
[2012-12-31 10:07:51] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:UserProfile:index.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 9238 [] []
[2012-12-31 10:07:51] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 10:07:52] request.INFO: Matched route "_user_profile" (parameters: "_controller": "CloudPod\UserBundle\Controller\UserController::profileAction", "id": "1", "_route": "_user_profile") [] []
[2012-12-31 10:07:53] security.DEBUG: Read SecurityContext from the session [] []
[2012-12-31 10:07:53] security.DEBUG: Reloading user from user provider. [] []
[2012-12-31 10:07:54] security.DEBUG: Username "" was reloaded from user provider. [] []
[2012-12-31 10:07:54] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:UserProfile:index.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 9238 [] []
[2012-12-31 10:07:54] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 10:07:57] request.INFO: Matched route "_user_profile" (parameters: "_controller": "CloudPod\UserBundle\Controller\UserController::profileAction", "id": "1", "_route": "_user_profile") [] []
[2012-12-31 10:07:58] security.DEBUG: Read SecurityContext from the session [] []
[2012-12-31 10:07:58] security.DEBUG: Reloading user from user provider. [] []
[2012-12-31 10:07:59] security.DEBUG: Username "" was reloaded from user provider. [] []
[2012-12-31 10:07:59] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:UserProfile:index.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 9238 [] []
[2012-12-31 10:07:59] security.DEBUG: Write SecurityContext in the session [] []
[2012-12-31 10:09:18] request.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 5110 [] []
[2013-01-04 19:29:55] request.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 5110 [] []
[2013-01-04 19:32:04] request.INFO: Matched route "_security_login" (parameters: "_controller": "CloudPod\UserBundle\Controller\SecurityController::loginAction", "_route": "_security_login") [] []
[2013-01-04 19:32:04] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2013-01-04 19:32:05] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:Forms:login.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\vendor\twig\twig\lib\Twig\Template.php line 280 [] []
[2013-01-04 19:32:05] security.DEBUG: Write SecurityContext in the session [] []
[2013-01-04 19:32:10] request.INFO: Matched route "_security_login" (parameters: "_controller": "CloudPod\UserBundle\Controller\SecurityController::loginAction", "_route": "_security_login") [] []
[2013-01-04 19:32:10] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2013-01-04 19:32:10] request.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Route "_welcome" does not exist.") in "CloudPodUserBundle:Forms:login.html.twig". (uncaught exception) at C:\wamp\www\CloudPod\vendor\twig\twig\lib\Twig\Template.php line 280 [] []
[2013-01-04 19:32:10] security.DEBUG: Write SecurityContext in the session [] []
[2013-01-04 19:38:19] request.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at C:\wamp\www\CloudPod\app\cache\prod\classes.php line 5110 [] []
